src/src/ex7/client/src/redux/actions/ViewsActions.js

```diff
@@ -0,0 +1,26 @@
+import actionTypes from "./constants";
+const showLoader = () => ({
+    type: actionTypes.SHOW_LOADER,
```

Comment:
can be unified to one action called `toggleLoader` and pass a parameter called "show" (boolean - true/false) to indicate if should be shown or hidden